apps/api-reference/src/evm-networks.ts

```diff
@@ -1,11 +1,47 @@
+import * as EvmChains from "@pythnetwork/contract-manager/data/chains/EvmChains.json";
+import * as EvmPriceFeedContracts from "@pythnetwork/contract-manager/data/contracts/EvmPriceFeedContracts.json";
+
+type EvmChains = {
+  id: string;
+  mainnet: boolean;
+  rpcUrl: string;
+  networkId: number;
+  type: string;
+  nativeToken?: string;
```

Comment:
I don't think you need to worry about adding types for fields that aren't in use with this app.  Typescript objects are not closed (meaning they allow undeclared fields to be present), and only including fields you actually need allows typescript to be more helpful if we wanted to e.g. restructure the shape of `EvmChains` (typescript would be able to tell us if we've restructured things in a way that's breaking anything in use)